Dockerfile-grocy-backend

```diff
@@ -16,31 +18,40 @@ RUN set -eux; \
 # Install build-time dependencies
 RUN     apk update && \
         apk add --no-cache \
-            composer \
             git \
             gnupg \
             wget
 
 # Install system dependencies
 RUN     apk add --no-cache \
-            php7-ctype \
-            php7-fpm \
-            php7-exif \
-            php7-fileinfo \
-            php7-gd \
-            php7-iconv \
-            php7-json \
-            php7-ldap \
-            php7-pdo_sqlite \
-            php7-simplexml \
-            php7-tokenizer
+            php8 \
+            php8-ctype \
+            php8-fpm \
+            php8-exif \
+            php8-fileinfo \
+            php8-gd \
+            php8-iconv \
+            php8-ldap \
+            php8-pdo_sqlite \
+            php8-simplexml \
+            php8-tokenizer \
+            php8-phar \
+            php8-curl \
+            php8-mbstring \
+            php8-openssl \
+            php8-zip \
+            php8-intl && \
+        ln -s /usr/bin/php8 /usr/bin/php && \
+        wget https://getcomposer.org/download/${COMPOSER_VERSION}/composer.phar -O /tmp/composer.phar && \
+        echo "${COMPOSER_CHECKSUM}  /tmp/composer.phar" | sha256sum -c - && \
```

Comment:
Spacing looks good here - I didn't realize until today that the second character after the hash value can be significant, and can indicate the input mode used for the checksum commands.